.travis.yml

```diff
@@ -1,8 +1,22 @@
-  language: python
-  python:
-    - 3.5
-    - 3.6
-  install:
-    - make
-  script:
-    - make test
+language: python
+
+matrix:
+  include:
+    - os: linux
```

Comment:
Keep in mind that the current CLI supports Windows which is not supported by Travis CI.

Reply:
That's right, I will take a look at https://www.appveyor.com in the next steps.